### PR TITLE
feat(schemas): Copy existing schemas to separate repo

### PR DESCRIPTION
**Pull request by kpears201**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "@babel/core": "^7.14.6",
     "@babel/node": "^7.14.5",
     "@babel/preset-env": "^7.14.5",
-    "@firebolt-js/openrpc": "1.2.0-beta.3",
+    "@firebolt-js/openrpc": "file:../firebolt-openrpc",
```
Comment (kpears201) [inline package.json:38]: Will update once release available

**Comment by cletustboone** [comment]
Is this moving more json out of the core sdk code and into the tooling repo?

**Comment by cletustboone** [comment]
Ref https://github.com/rdkcentral/firebolt-openrpc/pull/18

**Review by cletustboone — CHANGES_REQUESTED** [review]
I really think we need to have a discussion about where this stuff should live. The more I am looking at this, I think we need to move all of the JS template code here. The webpack job (i.e. the thing that packages the code for npm) should also live here. Then we use the tooling and run it on this repo as the target. There are several pretty big issues I'm running into with both of these things living in rdkcentral/firebolt-openrpc, which I'm happy to discuss. Would also like a refresher on the rationale for moving them.

**Comment by jlacivita** [comment]
There's two other PRs combined with this one that address at least a few of your concerns:

https://github.com/rdkcentral/firebolt-schemas/pull/2
https://github.com/rdkcentral/firebolt-openrpc/pull/18/files
https://github.com/rdkcentral/firebolt-core-sdk/pull/11

They move the following:

- shared schemas from `rdkcentral/firebolt-openrpc` into `rdkcentral/firebolt-schemas`
- core schemas from `rdkcentral/firebolt-core-sdk` into `rdkcentral/firebolt-schemas` so other SDKs may use them
- add `rdkcentral/firebolt-schemas` as a dependency for `firebolt-core-sdk` so it can use them

I agree about the JS template code, but we can solve that in a separate PR.

File: src/modules/parameters.json [hunk 2]
```diff
@@ -30,7 +30,9 @@
                     "navigateTo": {
                         "action": "entity",
                         "data": {
-                            "entityId": "abc"
+                            "entityId": "abc",
+                            "entityType": "program",
```
Comment (jlacivita) [inline src/modules/parameters.json:34]: Why did we need changes here, if we're just moving files?

Comment (kpears201) [reply src/modules/parameters.json:34]: The intents.json schema was actually duplicated in two different locations in openrpc and here. They must've differed a little bit. But I think these fields it was missing were meant to be required.

Comment (jlacivita) [reply src/modules/parameters.json:34]: that sounds right, IIRC. okay, thanks.

**Review by cletustboone — DISMISSED** [review]
Okay, got it. Standing down...

File: package.json [hunk 3]
```diff
@@ -35,11 +35,12 @@
...
     "ajv": "^6.12.6",
     "jest": "^27.0.4",
     "webpack": "^5.39.1",
-    "webpack-cli": "^4.2.0"
+    "webpack-cli": "^4.2.0",
+    "@firebolt-js/schemas": "file:../firebolt-schemas"
```
Comment (jlacivita) [inline package.json:43]: don't forget to update this one, too, once we make a release.

**Review by jlacivita — APPROVED** [review]
LGTM, see my comment about releasing `@firebolt-js/schemas` to npm

**Pull request merged**